src/modules/GetList.js

```diff
@@ -0,0 +1,19 @@
+async function getList() {
+  const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Zl4d7IVkemOTTVg2fUpp/scores/';
+  const response = await fetch(url);
+  const reply = await response.json();
+  const scoreListContainer = document.querySelector('.score-list-container');
+  scoreListContainer.innerHTML = '';
+  const sorted = reply.result.sort((a, b) => a.score - b.score);
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
- [ ] Kindly enable the linters and instead work on fixing the linter errors 🙏🏻. It helps with keeping consistent code standards